rubin_sim/skybrightness/interp_components.py

```diff
@@ -381,8 +381,11 @@ def __init__(self, mags=False, dark_sky_mags=None, fit_results=None):
             bp_temp.resample_bandpass(
                 wavelen_min=self.solar_spec.wavelen.min(),
                 wavelen_max=self.solar_spec.wavelen.max(),
-                wavelen_step=0.2,
+                wavelen_step=self.solar_spec.wavelen[1] - self.solar_spec.wavelen[0],
```

Comment:
The problem isn't here, it's that wavelen_min and wavelen_max should be within the range of the solar spectrum you're using. The step being 0.2 is fine.